proposed/container.md

```diff
@@ -32,7 +32,7 @@ Users of dependency injections containers (DIC) are referred to as `user`.
   While `ContainerInterface` only defines one mandatory parameter in `get()`, implementations
   MAY accept additional optional parameters.
 
-- `has` takes one unique parameter: an entry identifier. It MUST return `true`
+- `has` takes one unique parameter: an entry identifier, that MUST be a string. `has` MUST return `true`
```

Comment:
`s/that/which/`: "an entry identifier, *which* MUST be a string." Otherwise, LGTM.